docs/technical-reference.md

```diff
@@ -437,6 +437,97 @@ similar to the following to the KMS key policy:
 }
 ```
 
+### S3 buckets with SSE-KMS
+
+In order for Quilt to access and index buckets encrypted with SSE-KMS, you must do three things:
+
+1. Add KMS Key Usage to Quilt Permission Boundary
+2. Add Quilt Principals to KMS Key Policy
+3. Add KMS Key Access to a Scoure=Quilt Role
+
+NOTE: This will not work with the default Source=Custom Roles.
+
+#### 1. Add KMS Key Usage to Quilt Permission Boundary
+
+By default, AWS does not allow anything in your account to access KMS. If you haven't done so already,  create an IAM policy that explicitly enables KMS access.
+
+```json
+{
+  "Version": "2012-10-17",
+  "Statement": {
+    "Effect": "Allow",
+    "Action": [
+      "kms:Encrypt",
+      "kms:Decrypt"
+    ],
+    "Resource": "arn:aws:kms:us-west-2:111122223333:key/*"
+  }
+}
+```
+
+Go to CloudFormation > Your Quilt Stack -> Update -> Parameters and add the ARN of that IAM policy to  `ManagedUserRoleExtraPolicies` at the bottom of the page:
+
+![](../imgs/ManagedUserRoleExtraPolicies.png)
+
+If other policies are already in that field, you will need to add a comma before appending the ARN.
```

Comment:
This should be highlighted/add emphasis with a `>` at the beginning

Reply:
Propose we ignore so we can get this merged.